### PR TITLE
feat: add appimage links to download page

### PR DESCRIPTION
**Pull request by robertgzr**
Signed-off-by: Robert Günzler <r@gnzler.io>

### CODE REVIEW

**Comment by robertgzr** [comment]
image

**Comment by robertgzr** [comment]
the alignment in the bottom cards is a bit unfortunate :p but i didn't see an immediate way to fix that

File: src/pages/fiddle/index.tsx [hunk 1]
```diff
@@ -30,12 +30,21 @@ export default function FiddlePage() {
...
+      },
+      rpm: {
+        x64: `https://github.com/electron/fiddle/releases/download/${version.raw}/electron-fiddle-${version.version}-1.x86_64.rpm`,
+        arm64: `https://github.com/electron/fiddle/releases/download/${version.raw}/electron-fiddle-${version.version}-1.arm64.rpm`,
+        arm7l: `https://github.com/electron/fiddle/releases/download/${version.raw}/electron-fiddle-${version.version}-1.armv7hl.rpm`,
+      },
+      appimage: {
+        x64: `https://github.com/electron/fiddle/releases/download/${version.raw}/electron-fiddle-${version.version}-x86_64.AppImage`,
+        arm64: `https://github.com/electron/fiddle/releases/download/${version.raw}/electron-fiddle-${version.version}-arm64.AppImage`,
+        arm7l: `https://github.com/electron/fiddle/releases/download/${version.raw}/electron-fiddle-${version.version}-armv7hl.AppImage`,
```
Comment (dsanders11) [inline src/pages/fiddle/index.tsx:46]: These are not correct, see the artifacts under https://github.com/electron/fiddle/releases/tag/v0.33.0. The format for the AppImage filenames is `Electron.Fiddle-${version.version}-arm64.AppImage`.

Comment (robertgzr) [reply src/pages/fiddle/index.tsx]: :facepalm:

**Review by erickzhao — APPROVED** [review]

**Pull request merged**